### DISCUSSION TITLE
When to support Avalonia?

### DISCUSSION THREAD

**Discussion by yozochen** [cefsharp/CefSharp#4932] [General]
Avalonia
- https://github.com/AvaloniaUI/Avalonia
- https://avaloniaui.net/

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/discussions/3438#discussioncomment-464830

https://github.com/cefsharp/CefSharp.OutOfProcess has an example

If someone from the community wants to take lead then we can create a repo under https://github.com/cefsharp

I don't have any plans currently to invest time in this.
